### DISCUSSION TITLE
show number of players connected to a server

### DISCUSSION THREAD

**Discussion by 200779b** [pelican-dev/panel#123] [Ideas]
both in the server lists and in the console page, would be nice to display the number of players connected

**Comment by TekExplorer** [comment]
A player list? This is somewhat complicated.

If its a publicly accessible server, and the game has a way to detect that anonymously, it could be done purely on the front end

Otherwise, it may be necessary to do something on the node level.

Perhaps a program that can connect to the relevant container, query it, and report the data back to a plugin?

This could possibly have extra use as a ban list plugin.

I might give it a poke, maybe.

**Comment by 200779b** [comment]
Actually I was thinking about just a number, like how it says "Uptime: 2h09min", it could also say "Players online: 12".

**Reply by TekExplorer** [reply]
That still requires pinging the server itself. how is it supposed to know otherwise? might as well get the rest of the data while we're there for those who want it eh?

**Comment by parkervcp** [comment]
In theory https://github.com/Austinb/GameQ can potentially handle this. It was called out here as well https://github.com/pterodactyl/panel/issues/1218

**Reply by TekExplorer** [reply]
Its certainly one way of doing it, though it wont work on internal servers like bungeecord servers

not necessarily a deal breaker, but it is something to take into account.

**Comment by Mutex21** [comment]
Well, this one could be implemented as a paid add-in and fully supported by Pelican team, not integrated in Pelican prod.

**Comment by lancepioch** [comment]
This is a great idea for a future plugin!
